rkyv/Cargo.toml

```diff
@@ -45,13 +45,10 @@ copy_unsafe = []
 size_16 = []
 size_32 = []
 size_64 = []
-std = ["alloc", "bytecheck/std", "ptr_meta/std", "rend/std"]
+std = ["alloc", "bytecheck?/std", "ptr_meta/std", "rend?/std", "uuid?/std"]
 strict = ["rkyv_derive/strict"]
+uuid = ["dep:uuid", "bytecheck?/uuid"]
```

Comment:
This looks much better!

Reply:
I had to go check the new cargo resolvers reference. I was pretty sure that there was some way to get this to work in feature resolver v2.